client/src/containers/Workspaces/Workspaces.js

```diff
@@ -1,56 +1,51 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import useFetch from '../../hooks/useFetch';
+import { useAppContext } from '../App/AppContext';
 
 import Container from '../../components/Container/Container';
```

Comment:
We have single point importing implemented. You import multiple components from a single source `../../components`;